### PR TITLE
Remove span exporter from ConsoleExporter

### PR DESCRIPTION
**Pull request by cijothomas**
Make ConsoleExporter operate exclusively on `Activity` instead of `Span`, as `Activity` represent the OpenTelemetry `Span`
Related changes in console example, renaming as well.

### Checklist
- [ ] I ran Unit Tests locally.

For significant contributions please make sure you have completed the following items:

- [ ] Design discussion issue #
- [ ] Changes in public API reviewed

The PR will automatically request reviews from [code owners](https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/CODEOWNERS#L15) and trigger CI build and tests.

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs [hunk 1]
```diff
@@ -15,35 +15,101 @@
...
+                }
+                else
+                {
+                    System.Console.WriteLine("Activity ID - " + activity.Id);
+                    if (!string.IsNullOrEmpty(activity.ParentId))
+                    {
+                        System.Console.WriteLine("Activity ParentId - " + activity.ParentId);
+                    }
+
+                    System.Console.WriteLine("Activity OperationName - " + activity.OperationName);
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs:65]: Curious, do we expect synchronization (e.g. having the entire `else` block running mutual exclusively)?

File: src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs [hunk 2]
```diff
@@ -15,35 +15,101 @@
...
+                    if (resource != Resource.Empty)
+                    {
+                        System.Console.WriteLine("Resource associated with Activity");
+                        foreach (var resourceAttribute in resource.Attributes)
+                        {
+                            System.Console.WriteLine($"\t {resourceAttribute.Key} : {resourceAttribute.Value}");
+                        }
+                    }
+
+                    System.Console.WriteLine("\n");
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs:111]: Curious, do we prefer "\n" or [`System.Environment.NewLine`](https://docs.microsoft.com/en-us/dotnet/api/system.environment.newline) or two `System.Console.WriteLine()`?

File: src/OpenTelemetry.Exporter.Console/ConsoleExporterOptions.cs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// <copyright file="ConsoleExporterOptions.cs" company="OpenTelemetry Authors">
+﻿// <copyright file="ConsoleExporterOptions.cs" company="OpenTelemetry Authors">
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Console/ConsoleExporterOptions.cs:1]: Is there a BOM change?

File: src/OpenTelemetry.Exporter.Console/ConsoleExporterOptions.cs [hunk 4]
```diff
@@ -18,6 +18,6 @@ namespace OpenTelemetry.Exporter.Console
 {
     public class ConsoleExporterOptions
     {
-        public bool Pretty { get; set; }
+        public bool DisplayAsJson { get; set; }
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Console/ConsoleExporterOptions.cs:21]: Thanks for the clarification here 👍

File: src/OpenTelemetry.Exporter.Console/README.md [hunk 5]
```diff
@@ -1,21 +1,7 @@
 # OpenTelemetry.Exporter.Console
 
-This is a simple exporter that that JSON serializes collected spans and prints them to the Console and is intended to be used during learning how spans are creating and exported.
+This is a simple exporter that that JSON serializes collected activities and prints them to the Console and is intended to be used during learning how activities are created and exported.
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Console/README.md:3]: Not blocking this PR, should we use "activities" or "[Activity](https://docs.microsoft.com/dotnet/api/system.diagnostics.activity) objects" across all the comments and documents?

Comment (cijothomas) [reply src/OpenTelemetry.Exporter.Console/README.md:3]: Agree we need a consistent way.

Something like this

docs:
[Activity](https://docs.microsoft.com/dotnet/api/system.diagnostics.activity) objects (Activity is the .NET implementation of OpenTelemetry [Span](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#span))

Code comments:
Activity objects.

File: src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs (hunk 2 above)
Comment (cijothomas) [reply src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs]: I don't have preference, but i'd avoid the "\n".

File: src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs (hunk 1 above)
Comment (cijothomas) [reply src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs]: Not sure I understood the question. Can you clarify

Comment (reyang) [reply src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs]: If multiple threads are triggering this piece of code (e.g. using simple exporter/processor), do we expect the console output to be mixed up?

Comment (cijothomas) [reply src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs]: Got it. Yes i think it'll mix up the outputs. Its okay (?) as console is just for learning how activity looks like and we can keep it very simple.

**Pull request merged**